tests/test_exceptions.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import print_function
+
+import pytest
+import apypie
+
+
+def test_printable(capsys):
+    test_message = 'this is a printable message'
+    printable = apypie.exceptions.Printable()
+    printable.message = test_message
+    printable.print()
+    captured = capsys.readouterr()
+    assert captured.out == '{}\n'.format(test_message)
+
+
+def test_docloadingerror(capsys):
+    test_message = 'this is DocLoadingError'
+    test_error = apypie.exceptions.DocLoadingError(test_message)
+    with pytest.raises(apypie.exceptions.DocLoadingError) as e:
+        raise test_error
+        assert e.message == test_message
```

Comment:
Shouldn't this be outside of the `with` context?

Reply:
probably, yeah

Reply:
well, no, `e` is not defined outside the `with`

Reply:
It is because Python doesn't clean its scope. Because of the raise this is unreachable code now.

Reply:
ah, me--

Reply:
updated!